readthedocs/templates/core/project_detail_right.html

```diff
@@ -33,8 +33,8 @@ <h3>{% trans "Last Built" %}</h3>
   </p>
 {% endblock %}
 
-{% block owners %}
-    <h3>{% trans "Owners" %}</h3>
+{% block maintainers %}
```

Comment:
In case we change the block name, we will need to update our template in the corporate site:

    readthedocsinc/corporate/templates/core/project_detail_right.html